packages/core/src/render3/util/global_utils.ts

```diff
@@ -33,6 +33,30 @@ import {getDependenciesFromInjectable, getInjectorMetadata, getInjectorProviders
  * */
 export const GLOBAL_PUBLISH_EXPANDO_KEY = 'ng';
 
+const globalUtilsFunctions = {
+  'ɵgetDependenciesFromInjectable': getDependenciesFromInjectable,
+  'ɵgetInjectorProviders': getInjectorProviders,
+  'ɵgetInjectorResolutionPath': getInjectorResolutionPath,
+  'ɵgetInjectorMetadata': getInjectorMetadata,
+  /**
+   * Warning: this function is *INTERNAL* and should not be relied upon in application's code.
+   * The contract of the function might be changed in any release and/or the function can be
+   * removed completely.
+   */
+  'ɵsetProfiler': setProfiler,
+  'getDirectiveMetadata': getDirectiveMetadata,
```

Comment:
nit: a small proposal to move the comment above all internal functions:
```suggestion
  /**
   * Warning: functions that start with `ɵ` are considered *INTERNAL* and should not be relied upon
   * in application's code. The contract of those functions might be changed in any release and/or a function
   * can be removed completely.
   */
  'ɵgetDependenciesFromInjectable': getDependenciesFromInjectable,
  'ɵgetInjectorProviders': getInjectorProviders,
  'ɵgetInjectorResolutionPath': getInjectorResolutionPath,
  'ɵgetInjectorMetadata': getInjectorMetadata,
  'ɵsetProfiler': setProfiler,
  
  'getDirectiveMetadata': getDirectiveMetadata,
```